components/job-orchestration/job_orchestration/scheduler/job_config.py

```diff
@@ -47,4 +47,12 @@ class SearchConfig(BaseModel):
     end_timestamp: typing.Optional[int] = None
     ignore_case: bool = False
     path_filter: typing.Optional[str] = None
+    network_address: typing.Optional[typing.Tuple[str, int]] = None
     aggregation_config: typing.Optional[AggregationConfig] = None
+
+    @validator("network_address")
+    def validate_network_address(cls, field):
+        if field is not None and (field[1] < 0 or field[1] > 65535):
```

Comment:
```suggestion
        if field is not None and (field[1] < 1 or field[1] > 65535):
```
Since port 0 is reserved as a wildcard port.